app/models/oldCartModel.py

```diff
@@ -26,11 +26,22 @@ def get_all_by_uid(uid):
 FROM OldCarts
 WHERE uid = :uid
 ORDER BY time_created DESC
-limit 3
 ''',
                               uid=uid)
         return [OldCartModel(*row) for row in rows]
 
+    @staticmethod
+    def get_recent_three_by_uid(uid):
+        rows = app.db.execute('''
+SELECT cid, uid, cart_name, time_created
+FROM OldCarts
+WHERE uid = :uid
+ORDER BY time_created DESC
+limit 3
```

Comment:
maybe adding a "See all" button

Reply:
So this can be a button on the homepage that links the user to /oldcarts so that they can see all the oldcarts they had?

Reply:
No, under the first three, if they click "See all" it will show a list of all the old carts they have instead of just the most recent three